isis/src/dev/apps/camdev/main.cpp

```diff
@@ -132,7 +132,7 @@ void IsisMain() {
 
   if(noCamera) {
     try {
-      proj = (TProjection *) icube->projection();
+      tproj = (TProjection *) icube->projection();
```

Comment:
these changes seem to be from your other PR

Reply:
Yeah, I closed that and merged that into this one. Also with a CHANGELOG notice, as you requested, for this. 

It is a small thing, hopefully no need to reopen the other one and edit the changelog for that too.